src/pages/dashboard/ui/ticket/TIcketConfirmPage.tsx

```
@@ -24,18 +69,28 @@ const TicketConfirmPage = () => {
+      ) : (
+        <p className="text-center text-gray-500">티켓 정보를 불러오는 중...</p>
+      )}
       {isModalOpen && (
         <EmailDeleteMoal
-          mainText="WOOACON 2024의 일반 티켓 2매 구매를 취소하시겠습니까?. 취소 후에는 복구가 불가능합니다."
+          mainText={`${ticket?.title}의 ${ticket?.ticketName} ${orderIds.length}매 구매를 취소하시겠습니까?. 취소 후에는 복구가 불가능합니다.`}
           approveButtonText="티켓 취소"
           rejectButtonText="뒤로가기"
           onClose={() => setIsModalOpen(false)}
-          onClick={() => navigate('/menu/myticket')}
+          onClick={() => { cancleOrderTicket(orderIds),navigate('/menu/myticket') }}
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)